src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -60,6 +60,11 @@ function CardSection() {
         Navigation.resetToHome();
     }, []);
 
+    const viewPurchases = useCallback(() => {
+        const query = SearchUtils.buildQueryStringFromFilterValues({merchant: CONST.EXPENSIFY_MERCHANT});
```

Comment:
Not sure if we need to do that. @trjExpensify thoughts?

Reply:
I'm not sure about the status:paid tbh, because when we add the expense to their account we don't put it immediately on a `reimbursed` expense report. It's for'sure of type:expense though.

Reply:
I think we can start simple and update the filters if we're getting false positives